hostprocess/flannel/flanneld/flannel-overlay.yml

```diff
@@ -111,6 +111,10 @@ spec:
         - name: kube-proxy
           mountPath: /flannel-config-file
```

Comment:
The other issue here is that if flannel has moved to the `kube-flannel` ns and we update our deployments to do that then the kubeproxy and other kubeadm configs we were relying on can't be accessed since they live in kube-system. Will need to maybe create our configmaps or change the way those files are used.

Reply:
I looked into this a bit, The kubeproxy cm is used for the flannel connection to the api-server instead of using that we should use the token that get projected into the account. With containerd 1.6 if still requires so changes due to `CONTAINER_SANDBOX_MOUNT_POINT` and mounting but we can add a new cm to the deployment files that could be used. In the next version of containerd we can use in the cluster config that flannel provides

The kubeadm cm is used to get the pod and service cidrs.  This information pod cidr is already in `kube-flannel-cfg` in the ns `kube-flannel` in 0.17.  The service cider is something is static so we could just move it to the `kube-flannel-cfg-windows` that we create and read it from there.

This would remove the for the kube-adm and kube-proxy configmaps which out having to add the addtional RBAC rules to allow reading of those files.